lib/websockets.py

```diff
@@ -95,17 +95,18 @@ def run(self):
                 continue
             util.print_error('response', r)
             method = r.get('method')
-            params = r.get('params')
+            scripthash = r.get('params')[0]
             result = r.get('result')
             if result is None:
                 continue    
             if method == 'blockchain.scripthash.subscribe':
-                self.network.send([('blockchain.scripthash.get_balance', params)], self.response_queue.put)
+                self.network.subscribe_to_scripthash(
+                        scripthash, self.response_queue.put)
```

Comment:
This is a state machine; you missed what it is doing.
When receiving a reply to a `blockchain.scripthash.subscribe`, send a `blockchain.scripthash.get_balance` request.
Your change breaks it; it will never reach the next state.

Reply:
This was annoying me. :) Pushed a fix.